src/components/Calendar.vue

```diff
@@ -120,27 +120,48 @@ export default {
 	},
 };
 </script>
-
+<style scoped>
+.calendar {
+	background-color: white;
+	width: 387px;
+	box-shadow: 4px 4px 10px rgba(145, 149, 234, 0.3),
```

Comment:
This looks out of place. Should just be `box-shadow: 4px 4px 10px rgba(145, 149, 234, 0.3);`